Content.Server/GameTicking/Rules/SecretRuleSystem.cs

```diff
@@ -0,0 +1,51 @@
+using System.Collections.Generic;
+using System.Linq;
+using Robust.Shared.IoC;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Random;
+
+namespace Content.Server.GameTicking.Rules;
+
+public class SecretRuleSystem : GameRuleSystem
+{
+    [Dependency] private readonly IRobustRandom _random = default!;
+    [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+    [Dependency] private readonly GameTicker _ticker = default!;
+
+    public override string Prototype => "Secret";
+
+    private readonly IReadOnlyDictionary<string, float> _ruleTable = new Dictionary<string, float>
+    {
+        { "Traitor", 0.75f },
+    };
```

Comment:
Looks like this only gets converted to a list anyway so may as well just make it a list of tuples?